soda_machine.rb

```diff
@@ -7,12 +7,20 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.length
   end
 
   def find_soda(soda_brand)
+    @sodas.find { |soda| soda.brand == soda_brand }
   end
 
   def sell(soda_brand)
+    if find_soda(soda_brand) != nil
```

Comment:
Solid. Any way to make the `delete` and `find_soda` calls into a one-liner and not make two calls to `find_soda`?